tools/FileTestDefs.sh

```diff
@@ -296,6 +297,7 @@ license() {
               'docs/config/footer.html' \
               'docs/config/header.html' \
               'docs/config/layout.xml' \
+              'docs/config/MathJax.js$' \
```

Comment:
You can just add the license, js files support comments just fine.

Reply:
The reason I didn't is that since we didn't write the code and just got it from the conversion tool I felt we can't really stick our license on it :)

Reply:
Do we have to add/link to the license of the conversion tool?

Reply:
https://mathjax.github.io/MathJax-demos-web/convert-configuration/convert-configuration.html 

Not that I can tell